src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,26 @@
+import BootstrapButton from 'react-bootstrap/Button';
+import formatCreationDate from '../../../../helpers/formatCreationDate';
+
+export default function CourseCard(props) {
+	const course = props.course;
```

Comment:
You can perform direct props decomposition:
```suggestion
	export default function CourseCard({ course })
```

Please, take a look at JS destructuring operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment